components/project-list/project-list.jsx

```diff
@@ -66,15 +88,6 @@ export default React.createClass({
       return entries.sort((a,b) => {
         return bookmarkedIdArray.indexOf(a.id.toString()) > bookmarkedIdArray.indexOf(b.id.toString());
```

Comment:
Is this something you're using in a lot of places? The db returns these in numerical order by default, unfortunately so it's non-trivial to make it behave like this. That's not to say it's incredibly difficult, I'm just not sure it's worth  the lift here if there's a JS function that does what you need relatively easily.